pkg/api/router.go

```diff
@@ -38,6 +38,15 @@ func (s *server) setupRouting() {
 	handle(router, "/bytes", jsonhttp.MethodHandler{
 		"POST": http.HandlerFunc(s.bytesUploadHandler),
 	})
+
+	handle(router, "/bzz-file", jsonhttp.MethodHandler{
+		"POST": http.HandlerFunc(s.bzzFileUploadHandler),
+	})
+
+	handle(router, "/bzz-file/{addr}", jsonhttp.MethodHandler{
```

Comment:
We can call this endpoint `/files` right away as per https://hackmd.io/fGoeV1n2QficZwuHnYmb-g#File.